appveyor.yml

```diff
@@ -35,7 +35,7 @@ before_build:
 build_script:
     - cmd: |
         set GRADLE_OPTS="-Dorg.gradle.jvmargs='-XX:MaxMetaspaceSize=512m'"
-        gradlew.bat clean install check -Ptest.exclude="**/*15*" -Druntime=%RUNTIME% -DruntimeVersion=%RUNTIME_VERSION% --stacktrace --info --no-daemon
+        gradlew.bat clean install check -Ptest.exclude="**/*15*,**/Polling*,**/*estLoose*" -Druntime=%RUNTIME% -DruntimeVersion=%RUNTIME_VERSION% --stacktrace --info --no-daemon
```

Comment:
Curious why not **/TestLoose*? Did you leave the T off on purpose? Also, can you open an issue specifically listing the tests that this PR excluded and need to be investigated (and reference this PR in that issue)?

Reply:
For some reason, I seemed to have problems in the past getting it to filter correctly without the first wildcard. I can try again to make it cleaner.
#497 already lists the issues and shouldn't be auto-closed by this PR.